package.json

```diff
@@ -183,6 +183,15 @@
           "webpack"
         ]
       },
+      {
+        "packages": [
+          "@fluentui/babel-make-styles"
+        ],
+        "dependencies": [
+          "tslib",
+          "typescript"
```

Comment:
will syncpack still work with other dependencies once you declare a new version group ?

Reply:
Yes

![Syncpack warns about different TS versions](https://user-images.githubusercontent.com/14183168/115395917-b36fdc00-a1e4-11eb-8e23-5fe7c3efd581.png)